contrib/test/ci/system-packages.yml

```diff
@@ -63,11 +63,22 @@
       - rpcbind
       - rsync
       - sed
+      - policycoreutils
```

Comment:
This should not be necessary, since 
container-selinux requires policycoreutils.

```
# rpm -q container-selinux --requires
/bin/sh
/bin/sh
/bin/sh
/bin/sh
/bin/sh
libselinux-utils
policycoreutils
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsZstd) <= 5.4.18-1
sed
selinux-policy >= 37.12-2.fc37
selinux-policy-base >= 37.12-2.fc37
selinux-policy-targeted >= 37.12-2.fc37
```